packages/entity/src/EntityConfiguration.ts

```diff
@@ -7,7 +7,7 @@ import { mapMap, invertMap, reduceMap } from './utils/collections/maps';
  * The data storage configuration for a type of Entity. Contains information relating to IDs,
  * cachable fields, field mappings, and types of cache and database adapter.
  */
-export default class EntityConfiguration<TFields extends Record<string, any>> {
+export default class EntityConfiguration<TFields extends Omit<Record<string, any>, 'constuctor'>> {
```

Comment:
```suggestion
export default class EntityConfiguration<TFields extends Omit<Record<string, any>, 'constructor'>> {
```

Reply:
Lol nice catch. This was me testing https://github.com/expo/entity/pull/235#issuecomment-2148216997. Will remove.